src/main/java/com/google/sps/data/Email.java

```diff
@@ -14,18 +14,55 @@
 
 package com.google.sps.data;
 
+import com.google.sps.servlets.Post;
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+
+/** Creates and converts HTML templates for email sending. */
 public final class Email {
 
   public static final String welcomeSubject = "⭐ noms: welcome to your free food finder!";
-  public static final String dailyDigestSubject = "⭐ noms: daily digest of free food!";
+  public static final String newPostSubject = "⭐ noms: new free food near you!";
+
+  // TODO: Find relative path instead of absolute (FileNotFoundException if just ./WelcomeEmail.html).
+  public static final String welcomeContentPath = "/home/areeta/noms/src/main/java/com/google/sps/data/WelcomeEmail.html";
+  public static final String newPostPath = "/home/areeta/noms/src/main/java/com/google/sps/data/NewPost.html";
+
+  /**
+    * Convert HTML to String for MimeMessage to configure it into emails.
+    *
+    * @param path path of HTML file
+    * @return string version of HTML file
+    * @throws IOException
+    */
+  public static String getStringFromHTML(String path) throws IOException {
 
-  public static final String welcomeContent = "<body style=\"margin: 3rem\">" +
-    "<h1>welcome to noms</h1>" +
-    "<p> thank you so much for subscribing to our mailing list of free food!";
+    File HTMLfile = new File(path);
+    String str = FileUtils.readFileToString(HTMLfile, "utf-8");
+    return str;
+  }
 
-  public static final String dailyDigestContent = "<body style=\"margin: 3rem\">" +
-    "<h1>daily digest of free food @ university of california-irvine</h1>" +
-    "<div style=\"background: green; border-radius: 15px;" + 
-    "<p>pad thai @ aldrich park</p>";
+  /**
+    * Add new post information to emails.
+    *
+    * @param post new post object
+    * @return string of HTML file with new post
+    * @throws IOException
+    */
+  public static String addNewPost(Post post) throws IOException {
 
+    // TODO: Include AM/PM, date formatted, and time zone to be more user friendly.
+    String emailContent = getStringFromHTML(newPostPath);
+    emailContent = emailContent.replace("[organizationName]", post.organizationName);
+    emailContent = emailContent.replace("[location]", post.location);
+    emailContent = emailContent.replace("[month]",  Integer.toString(post.month));
+    emailContent = emailContent.replace("[day]", Integer.toString(post.day));
+    emailContent = emailContent.replace("[startHour]", Integer.toString(post.startHour));
+    emailContent = emailContent.replace("[startMinute]", Integer.toString(post.startMinute));
```

Comment:
I noticed in one of the preview images that times were displaying as "17:0" instead of "17:00", does that fall under the TODO above this block?

Reply:
Yes! Since 00 shortens to 0 in Java, I will have to manually change up the format!